docs/learn-governance.md

```diff
@@ -165,35 +175,35 @@ Polkadot introduces a concept "Adaptive Quorum Biasing", which functions as a le
 
 Let's use the above image as an example.
 
-If there is publicly submitted referenda only has 25% turnout, the tally of "aye" votes has to reach 66% for it to pass since we applied the ``Positive Turnout Bias``.
+If a publicly submitted referenda only has 25% turnout, the tally of "aye" votes has to reach 66% for it to pass since we applied `Positive Turnout Bias`.
```

Comment:
```suggestion
If a publicly submitted referendum only has 25% turnout, the tally of "aye" votes has to reach 66% for it to pass since we applied `Positive Turnout Bias`.
```